api/__tests__/all.test.js

```diff
@@ -218,33 +241,98 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(response.body.msg).toEqual("Username not found");
       });
   });
-  test("Extra properties in post object are ignored", () => {
+
+  test("if username not found in the database return correct error", () => {
     return request(app)
       .post("/api/articles/4/comments")
-      .send({
-        username: "lurker",
-        body: "weird article",
-        extraProp: "extraValue",
-      })
-      .expect(201)
+      .send({ username: "unknownUser", body: "good article" })
+      .expect(404)
       .then((response) => {
-        const commentObject = response.body.comment;
-        const expectedCommentObject = {
-          author: "lurker",
-          body: "weird article",
+        expect(response.body.msg).toEqual("Username not found");
+      });
+  });
+});
```

Comment:
We have a lot of code here not relevant to the current task. Please make sure that all code on a branch is only relevant to the task you're working on. It makes it hard for us to review and means that certain code won't be reviewed as we look for the relevant code.